dvc/state.py

```diff
@@ -32,6 +32,8 @@ def __init__(self, dvc_version, expected, actual):
 
 
 class StateBase(object):
+    # state_file="state"
+    # temp_files=[]
```

Comment:
This is reasonable :) 
```suggestion
    temp_files=[]
```